RedHatInsights/insights-inventory-frontend
# feat(ESSNTL-3724): Add /groups route & empty state

Implements https://issues.redhat.com/browse/ESSNTL-3724.

This adds a new route /groups that leads to the groups table page. Now the page defaults to empty state in all cases and doesn't perform any network requests.

## How to test

Navigate to /insights/inventory/groups; check the header and empty state are rendered. Buttons and links are not expected to work at the moment.

## Screenshots

![Screenshot 2023-02-07 at 15-50-25 Inventory Groups Red Hat Insights console redhat com](https://user-images.githubusercontent.com/31385370/217279406-9b86e161-4f2d-42c7-bba0-3000ff232593.png)
